src/silo/preprocessing/preprocessor.test.cpp

```diff
@@ -114,14 +285,19 @@ const Scenario EMPTY_INPUT_NDJSON = {
          }
       }
    )",
-   .expected_query_result = nlohmann::json::parse(R"(
+      .expected_query_result = nlohmann::json::parse(R"(
 [])")
+   }
 };
 
-const Scenario EMPTY_INPUT_NDJSON_UNPARTITIONED = {
-   .input_directory = "testBaseData/emptyInputNdjsonUnpartitioned/",
-   .expected_sequence_count = 0,
-   .query = R"(
+const Scenario<Success> EMPTY_INPUT_NDJSON_UNPARTITIONED = {
+   .test_name = "EMPTY_INPUT_NDJSON_UNPARTITIONED",
+   .input_data = []() { return std::vector<NdjsonInputLine>{}; },
+   .database_config = NDJSON_WITH_NUMERIC_NAMES.database_config,
+   .reference_genomes = NDJSON_WITH_NUMERIC_NAMES.reference_genomes,
```

Comment:
IMO this is a little too sneaky. I wouldn't expect that someone reuses some random test data somewhere else. If someone adds `partition_by` to the `NDJSON_WITH_NUMERIC_NAMES.database_config`, then this test is also affected. I would rather split them:
* I'm not sure about the database config. Does it make sense to have a `DEFAULT_DATABASE_CONFIG_WITHOUT_PARTITIONING` here? Would other tests reuse it?
* It probably makes sense to have a standard `DEFAULT_REFERENCE_GENOMES` here that several tests can reuse (mostly tests than don't really care about which sequences are available).